### PR TITLE
Deconstruction_Recipe_Style_3

### PR DESCRIPTION
**Pull request by karols146**
Just next part Deconstruction Style clean.

### CODE REVIEW

**Comment by karols146** [comment]
@mugling Can you check this if you have some time, i would like to merge next part. 
Thanks.

File: data/json/recipes/recipe_deconstruction.json [hunk 1]
```diff
@@ -590,435 +590,352 @@
...
+    "type": "recipe",
+    "id_suffix": "uncraft",
+    "category": "CC_NONCRAFT",
+    "subcategory": "CSC_NONCRAFT",
+    "skill_used": "electronics",
+    "skills_required": [ "firstaid", 5 ],
+    "difficulty": 7,
+    "time": 50000,
+    "reversible": true,
+    "qualities": [ {"id": "SCREW","level": 1,"amount": 1 } ],
```
Comment (illi-kun) [inline data/json/recipes/recipe_deconstruction.json]: amount = 1 is a default value, so it's possible to remove ```,"amount": 1``` entirely.

File: data/json/recipes/recipe_deconstruction.json [hunk 2]
```diff
@@ -590,435 +590,352 @@
...
```
Comment (illi-kun) [inline data/json/recipes/recipe_deconstruction.json]: ... the same here and below.

**Comment by karols146** [comment]
@illi-kun should be ok now.

**Pull request merged**